lib/node/fullnode.js

```diff
@@ -151,9 +153,36 @@ class FullNode extends Node {
       port: this.config.uint('http-port'),
       apiKey: this.config.str('api-key'),
       noAuth: this.config.bool('no-auth'),
-      cors: this.config.bool('cors')
+      cors: this.config.bool('cors'),
+      maxTxs: this.config.uint('max-txs')
     });
 
+    // Indexers
+    if (this.config.bool('index-tx')) {
+      this.txindex = new TXIndexer({
+        network: this.network,
+        logger: this.logger,
+        blocks: this.blocks,
+        chain: this.chain,
+        prune: this.config.bool('prune'),
+        memory: this.config.bool('memory'),
+        prefix: this.config.filter('index').str('prefix') || this.config.prefix
```

Comment:
The second argument to `str` can be the fallback value

Reply:
I think the behavior is the same. However, perhaps it would be better to use `this.config.str('index-prefix')`, as the filter logic is somewhat cumbersome to work around, for example the wallet port config.

Reply:
Done.